src/main/ruby/truffleruby/core/io.rb

```diff
@@ -1221,6 +1222,12 @@ def prepare_read_string(str)
     @lineno += 1
   end
 
+  def path
+    @path.dup
+  end
+
+  alias_method :to_path, :path
```

Comment:
```suggestion
  end
  alias_method :to_path, :path
```
Let's keep them together that way it's easier to see what is aliased to what